cfbs/utils.py

```diff
@@ -278,6 +308,18 @@ def cfbs_dir(append=None) -> str:
     return os.path.join(directory, append)
 
 
+def string_sha256(input):
+    return hashlib.sha256(input.encode("utf-8")).hexdigest()
+
+
+def file_sha256(file):
+    h = hashlib.sha256()
+
+    h.update(open(file, "rb").read())
```

Comment:
Remember to `close()` the file afterwards. Or you can use the `with` keyword to do it automatically

Reply:
+1 for using context managers (`with`)